src/components/hook-form/RHFAutocomplete.tsx

```diff
@@ -0,0 +1,50 @@
+import { useFormContext, Controller } from 'react-hook-form';
+import { Autocomplete, TextField, TextFieldProps } from '@mui/material';
+
+type IProps = {
+  name: string;
+  options: string[];
+  freeSolo?: boolean;
+  width?: string | number;
+};
+
+type Props = IProps & TextFieldProps;
+
+export default function RHFTAutocomplete({
+  name,
+  options,
+  freeSolo = true,
+  width = '300px',
```

Comment:
I would use mui numbers, and avoid hardcoded px

Reply:
see comment below